cmd/internal/cli/actions_darwin.go

```diff
@@ -11,4 +11,5 @@ import (
 
 // TODO: Let's stick this in another file so that that CLI is just CLI
 func execStarter(cobraCmd *cobra.Command, image string, args []string, name string) {
```

Comment:
I would say, move the function `execStarter` here into `actions_unsupported.go` and have it be a `panic`.